tests/components/unifiprotect/test_light.py

```diff
@@ -0,0 +1,141 @@
+"""Test the UniFi Protect light platform."""
+# pylint: disable=protected-access
+from __future__ import annotations
+
+from copy import copy
+from unittest.mock import AsyncMock, Mock, patch
+
+import pytest
+from pyunifiprotect.data import Light
+
+from homeassistant.components.light import ATTR_BRIGHTNESS
+from homeassistant.components.unifiprotect.const import DEFAULT_ATTRIBUTION
+from homeassistant.const import (
+    ATTR_ATTRIBUTION,
+    ATTR_ENTITY_ID,
+    STATE_OFF,
+    STATE_ON,
+    Platform,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from .conftest import MockEntityFixture
+
+
+@pytest.fixture(name="light")
+async def light_fixture(
+    hass: HomeAssistant, mock_entry: MockEntityFixture, mock_light: Light
+):
+    """Fixture for a single light with only the button platform active, no extra setup."""
+
+    # disable pydantic validation so mocking can happen
+    Light.__config__.validate_assignment = False
+
+    light_obj = mock_light.copy(deep=True)
+    light_obj._api = mock_entry.api
+    light_obj.name = "Test Light"
+    light_obj.is_light_on = False
+
+    mock_entry.api.bootstrap.lights = {
+        light_obj.id: light_obj,
+    }
+
+    with patch("homeassistant.components.unifiprotect.PLATFORMS", [Platform.LIGHT]):
```

Comment:
Reminder about not patching PLATFORMS (ok to defer for the next PR)

Reply:
Yep, sorry about that. I missed the notification for your reply on the previous PR. Will do that now.